search-for-landmark/demo.html

```diff
@@ -20,16 +20,16 @@
         <h1>Search for a Landmark</h1>
         <p>Request the location of a landmark and display it on the map.</p>
     </div>
-    <p>This example makes a landmark search and retrieves the name and location of 
+    <p>This example makes a landmark search and retrieves the name and location of
       the <b>Eiffel Tower</b> based on the input text. A clickable marker is placed on the location found.</p>
     <div id="map"></div>
     <div id="panel"></div>
     <h3>Code</h3>
     <p>Access to the geocoding service is obtained from the <code>H.service.Platform</code>
-       by calling <code>getGeocodingService()</code>. The <code>discover()</code> method is used 
-       to find a landmark by passing in the relevant parameters as defined in 
-       <a href="https://developer.here.com/documentation/geocoding-search-api/dev_guide/topics/endpoint-discover-brief.html" target="_blank">Geocoding and Search API v7
-       </a>. 
+       by calling <code>getGeocodingService()</code>. The <code>discover()</code> method is used
```

Comment:
`discover` method is provided by `H.service.SearchService` (Geocoding&Search API v7) 

```suggestion
       by calling <code>getSearchService()</code>. The <code>discover()</code> method is used
```